mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -140,8 +184,8 @@ def ping():  # pylint: disable=unused-variable
     def transformation():  # pylint: disable=unused-variable
         """
         Do an inference on a single batch of data. In this sample server,
-        we take data as CSV or json, convert it to a Pandas DataFrame,
-        generate predictions and convert them back to CSV.
+        we take data as CSV or json, convert it to a Pandas DataFrame or Numpy,
+        generate predictions and convert them back to CSV/json.
```

Comment:
Looking at line 227 doesn't this only return json?

Reply:
good catch